.github/workflows/eks-tunnel.yaml

```diff
@@ -54,6 +54,11 @@ jobs:
           sudo tar xzvfC eksctl_$(uname -s)_amd64.tar.gz /usr/bin
           rm eksctl_$(uname -s)_amd64.tar.gz
 
+      - name: Install helm
+        uses: azure/setup-helm@18bc76811624f360dbd7f18c2d4ecb32c7b87bab
+        with:
+          version: v3.8.2
+
```

Comment:
I'm curious: why is that needed? Helm is already pre-installed on GH Runners: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md

Reply:
Scratch that, I just saw the commit description. In this case I would suggest

Reply:
Yup, however there is issue with latest helm version e.g. 3.9.0, so we need to use lower version.

Also, avoiding version drift is not a bad thing in general.

https://github.com/helm/helm/issues/10975